client/src/common/components/Toolbar/components/UserBar/UserBar.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import { logoutCurrentUser } from 'modules/authorization/model/actions';
+import { getCurrentUser } from 'modules/authorization/model/selectors';
+import { UserPropType } from 'common/constants/propTypes';
+import UserIcon from 'assets/images/user-solid.svg';
+import LogoutIcon from 'assets/images/sign-out.svg';
+
+class UserBar extends Component {
+  state = {
+    hideMenu: true
+  };
+
+  componentWillUnmount() {
+    this.removeEventListeners();
+  }
+
+  addEventListeners = () => {
+    document.addEventListener('click', this.hideMenu);
+    document.addEventListener('keydown', this.onPressEscape);
+  };
+
+  removeEventListeners = () => {
+    document.removeEventListener('click', this.hideMenu);
+    document.removeEventListener('keydown', this.onPressEscape);
+  };
+
+  hideMenu = () => {
+    this.setState({ hideMenu: true });
+    this.removeEventListeners();
+  };
+
+  showMenu = () => {
+    this.setState({ hideMenu: false });
+    this.addEventListeners();
+  };
+
+  onPressEscape = e => (e.code === 'Escape' ? this.hideMenu(e) : null);
```

Comment:
change this to basic to basic if statement
I know it's probably mine :) but update it here please :)